File: crates/fresh-editor/tests/fixtures/syntax_highlighting/hello.scala
object Hello {
  def greet(name: String): String = {
    s"Hello, $name!"
  }

  def main(args: Array[String]): Unit = {
    val message = greet("World")
    println(message)
    val items = List(1, 2, 3)
    items.foreach(i => println(s"Item: $i"))
  }
}

File: crates/fresh-editor/tests/fixtures/syntax_highlighting/hello.kt
// Kotlin syntax highlighting test
package hello

fun greet(name: String): String {
    return "Hello, $name!"
}

data class Config(
    val version: String,
    val enabled: Boolean = true,
    val count: Int = 42
)

fun main() {
    val message = greet("World")
    println(message)

    val items = listOf(1, 2, 3, 4, 5)
    items.filter { it > 2 }
         .forEach { println("Item: $it") }
}
